packages/tabs/src/component-tab.tsx

```diff
@@ -1,32 +1,20 @@
 import React from 'react';
 import { classNames as cn } from '@chbphone55/classnames';
-import { tab as c } from '@warp-ds/component-classes';
+import { tab as ccTab } from '@warp-ds/component-classes';
 import type { TabProps } from './props';
 
-const setup = ({
-  className,
-  isActive,
-  setActive,
-  contained,
-  ...rest
-}: any) => ({
-  tab: cn({
+const setup = ({ className, isActive, setActive, ...rest }: any) => ({
+  tab: cn([ccTab.tab], {
     [className]: !!className,
-    [c.tab]: true,
-    [c.tabActive]: isActive,
-    [c.tabContained]: contained,
-    [c.tabContainedActive]: contained && isActive,
+    [ccTab.tabActive]: isActive,
   }),
-  icon: cn({
-    [c.icon]: true,
-    [c.iconUnderlined]: !contained,
-    [isActive ? c.iconUnderlinedActive : c.iconUnderlinedInactive]: !contained,
+  icon: cn([ccTab.icon], [ccTab.iconUnderlined], {
+    [isActive ? ccTab.iconUnderlinedActive : ccTab.iconUnderlinedInactive]:
+      true,
```

Comment:
Minor, this could be refactored as 👇 Wdyt?
```suggestion
  icon: cn(ccTab.icon, ccTab.iconUnderlined, isActive ? ccTab.iconUnderlinedActive : ccTab.iconUnderlinedInactive)
```